src/modules/did/did.service.ts

```diff
@@ -495,4 +559,102 @@ export class DIDService implements OnModuleInit, OnModuleDestroy {
       this.logger.debug(inspect(jobsCounts, { depth: 2, colors: true }));
     }
   }
+
+  /**
+   * Finds DID's changed in given block interval. If fetching of DID's fails at some block, then further fetching stops.
```

Comment:
Very minor nit, but I wouldn't us the plural rather than possessive "s" here.
```suggestion
   * Finds DIDs changed in given block interval. If fetching of DIDs fails at some block, then further fetching stops.
```